deploy/test-environments/fleet_api/src/install_agentless_integrations.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python
+"""
+This script installs CSPM integrations on the 'Agentless' agent policy.
+
+The following steps are performed:
+1. Create a CSPM AWS integration.
+2. Create a CSPM Azure integration.
+3. Create a CSPM GCP integration.
+"""
+
+import os
+import configuration_fleet as cnfg
+from api.package_policy_api import create_cspm_integration
+from package_policy import (
+    generate_package_policy,
+    generate_policy_template,
+)
+from loguru import logger
+
+AGENT_POLICY_ID = "agentless"
+
+
+def generate_aws_integration_data():
+    """
+    Generate data for creating CSPM AWS integration
+    """
+    access_key_id = os.getenv("AWS_ACCESS_KEY_ID", "")
+    secret_access_key = os.getenv("AWS_SECRET_ACCESS_KEY", "")
+    return {
+        "name": "cspm_aws",
```

Comment:
May be add some randomization there. We have a function `generate_random_name` which is part of the `package_policy` module and can do that. You can use it like this:
`"name": generate_random_name("cspm_aws")`